.babelrc

```diff
@@ -1,5 +1,3 @@
 {
-	"optional": [
-		"runtime"
-	]
+  "presets": ["es2015", "stage-0"]
 }
```

Comment:
Babel 6 has an interesting issue with default exports in which it doesn't actually create a module.exports statement in the transpiled js.  Could you also install the package 'babel-plugin-add-module-exports' and add it as a plugin here?